Aztec/Classes/Extensions/NSAttributedString+Lists.swift

```diff
@@ -63,13 +63,82 @@ extension NSAttributedString
     func rangeOfTextList(atIndex index: Int) -> NSRange? {
         var effectiveRange = NSRange()
         let targetRange = rangeOfEntireString
-        guard let _ = attribute(TextList.attributeName, at: index, longestEffectiveRange: &effectiveRange, in: targetRange) as? TextList else {
+        guard
+            let paragraphStyle = attribute(NSParagraphStyleAttributeName, at: index, longestEffectiveRange: &effectiveRange, in: targetRange) as? ParagraphStyle,
+            let _ = paragraphStyle.textList
+        else {
             return nil
         }
 
         return effectiveRange
     }
 
+    /// Returns the range of the given text list that contains the given location.
+    ///
+    /// - Parameter list: The textList to search for.
+    /// - Parameter location: The location in the text list.
+    ///
+    /// - Returns: An NSRange optional containing the range of the list or nil if no list was found.
+    ///
+    func range(of list: TextList,
+               at location: Int) -> NSRange? {
```

Comment:
Can we have a single line for the method signature? (Unless this is an Aztec convention!)